2021/04.md

```diff
@@ -118,7 +118,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
 - Frank Yung-Fong Tang and Shane F. Carr will not be available 12:30-14:00 PM ET on Tuesday April 20th nor after 1:30PM ET on Wednesday April 21st. 
 - Shane F. Carr is unavailable Thursday after 13:00 ET; wants to be present for all Intl- and Regex-related topics.
 - Please schedule NVC Training Proposal after "Requests for services from Ecma for TC39"
-- Please schedule the “set notation + properties of strings update” somewhere within 12:00–14:00 ET on Thursday April 22nd so both champions and all stakeholders can attend.
+- Please schedule the “set notation + properties of strings update” from 11:00–11:30 ET on Thursday April 22nd so both champions can attend.
```

Comment:
was Thursday supposed to be updated ? maybe Wednesday?

Reply:
ooh just saw your message, requesting Tuesday morning. I'll work on it.

Reply:
I somehow managed to mess up a one-line change :D Good catch; fixed, and yes it was meant to be Tuesday indeed!